ui/desktop/electron-app/src/cli/path.js

```diff
@@ -20,6 +21,16 @@ module.exports = {
       : path.resolve(process.resourcesPath, 'cli', binaryName);
     const isBuiltInCli = existsSync(builtInCliPath);
 
+    // Filter out the current working directory from the list of binaries as spawn method picks up incorrect binary path
+    if (isWindows()) {
+      const binaries = which.sync(binaryName, { nothrow: true, all: true });
+
+      const filteredBinaryPath = binaries.filter(
+        (binary) => !binary.startsWith(process.cwd()),
+      );
+      const resolvedBinaryPath = path.resolve(filteredBinaryPath[0]);
```

Comment:
What does `path.resolve` do here? Isn't the `filteredBinaryPath[0]` already an absolute path?

Reply:
you are right, we don't need that.